Cargo.toml

```diff
@@ -1,5 +1,11 @@
 [workspace]
 
 members = [
+]
+
+# When doing `testground run`, the plan directory (e.g. smoke) is copied to the testground working
+# directory ($TESTGROUND_HOME/data/work/requests), and then the build starts. So each plan directory
+# should have its Cargo.lock.
+exclude = [
```

Comment:
Excluding the plan directory (`smoke`) from the workspace because of the reason I wrote here. Is it okay to remove this `Cargo.toml` as now there is no member in the workspace?